task8/Action.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Class is for choose he command
+ */
+public class Action {
+  /**
+   * Do chosen command and exit from program
+   *
+   * @param lists entered data of items
+   */
+  public void act(ArrayList<Item> lists) {
+    String flag;
+    Scanner sc = new Scanner(System.in);
+
+    do {
+      System.out.println("Enter command: count types or count all or average price or average price type or enter exit for exit:  ");
+      flag = sc.nextLine();
+      if (flag.equals("count types")) {
```

Comment:
Approach that is based on multiple if is not extendable. You must put your commands to container and iterate via it to choose the appropriate one